dagmc_stats.py

```diff
@@ -34,13 +36,11 @@ def get_dagmc_tags(my_core):
 def get_native_ranges(my_core, meshset, entity_types):
     """
     Get a dictionary with MOAB ranges for each of the requested entity types
-
```

Comment:
I'm not sure what PEP8 says about this (if anything) but I think docstrings are more readable with these blank lines in there.